tests/test_cylinder_model.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+from fenicsxconcrete.sensor_definition.other_sensor import ReactionForceSensorBottom
+from fenicsxconcrete.experimental_setup.concrete_cylinder import (
+    ConcreteCylinderExperiment,
+)
```

Comment:
why the parentheses?

Reply:
prob. more than 80 characters in a row. It's fine

Reply:
Sometimes `black` formatting is not the best, I have fixed that